### PR TITLE
David added as author + version upgrade to 0.6.9

### PR DESCRIPTION
**Pull request by lindsayplatt**
0.7.0 once we test and make any updates on repgen

### CODE REVIEW

**Comment by jordansread** [comment]
@lindsaycarr do you have some repgen coders lined up to turn up any new bugs this week?

**Comment by lindsayplatt** [comment]
that would be me - I'm working on that today

**Comment by jordansread** [comment]
@mhines-usgs too?

**Comment by jordansread** [comment]
would be good to get some new eyes on it that weren't part of the sprint

**Comment by lindsayplatt** [comment]
@mhines-usgs and I are testing w/ repgen. See #375 for details

**Comment by lindsayplatt** [comment]
I'm merging since only DESCRIPTION file updates (+ markdown file from previously merged pr #373 )

File: DESCRIPTION [hunk 1]
```diff
@@ -1,8 +1,8 @@
 Package: gsplot
 Type: Package
 Title: Geological Survey Plotting
-Version: 0.6.1
-Date: 2016-05-12
+Version: 0.6.9
+Date: 2016-07-05
 Authors@R: c( person("Jordan", "Read", role = "aut",
     email = "jread@usgs.gov"),
     person("Laura", "DeCicco", role = c("aut","cre"),
```
Comment (ldecicco-USGS) [inline DESCRIPTION:8]: I had just copied this from something originally, probably want to change @jread-usgs or @lindsaycarr to "cre"

**Comment by lindsayplatt** [comment]
Ha, right as I merged - will change with new PR

**Pull request merged**